HotKey/HotKey_MainFolder/Program.cs

```diff
@@ -16,7 +16,7 @@ static void Main()
         {
             Application.EnableVisualStyles();
             Application.SetCompatibleTextRenderingDefault(false);
-            Application.Run(new Form1());
+            Application.Run(new ModeForm());
```

Comment:
Maybe change from ModeForm to something more specific just for clarity. Took me a minute to figure out which this is.